rpc/jsonrpc.go

```diff
@@ -22,35 +38,47 @@ type (
 		Id      string          `json:"id"`
 	}
 
-	RPCResponse struct {
+	// RPCResponse MUST follow the JSON-RPC specification for Response object
+	// reference: http://www.jsonrpc.org/specification#response_object
+	RPCResponse interface{}
```

Comment:
How about making this an interface with a marker method `func AssertIsRPCResponse()`?